src/docs/files/devguide04.md

```diff
@@ -1,35 +1,34 @@
-
-#### 2. Once the auctions starts, the bidders get to start placing bids by setting the following parameters and submitting a transaction:
+Once the auctions starts, the bidders get to start placing bids. They need to:
 
 - Approved token (if first time)
 - Select amount of bid tokens willing to commit to the auction
 - Select maximum price willing to pay
+- Send the order placement transaction
 
-After submitting the transaction, the bidder’s bid will be valid, and the amount of bid tokens that he selected would be subtracted from his balance.
+After submitting the transaction, the bidder’s bid will be valid, and the amount of bid tokens that they selected would be subtracted from his balance.
 
-#### 3. Once the auction concludes, the auctioneer submits a transaction which will calculate the closing price. Closing price is calculated using the following method:
+Check the more [detailed guide](/docs/participate-as-a-bidder#topAnchor).
 
-1. Bids are gathered by the smart contract and sorted from highest to lowest price
-2. The smart contract works back from the highest bid, adding each bid’s amount of tokens to buy, until the sum multiplied by the bid's price equals the original amount to sell
-3. The price of this particular bid is selected as the final closing for all participants.
-4. Participants that selected a maximum price at the final closing price or higher receive tokens, the ones that had selected a maximum price below the closing price are left out
+_Important: as a bidder, it is important to note that many participants might submit new bids right before the auction closes, making it difficult to predict the closing price of the auction ahead of time. This, it is important for bidders to submit their bids with the highest price they are comfortable to pay for the asset_
 
-#### 4. After the price has settled, the auctioneer will receive the proceeds of the auction. In contracts the bidder will need to submit an additional transaction to claim their auction proceeds.
+#### 3. Auctioneer: On-chain price calculation
 
-_Important: as a bidder, it is important to note that many participants might submit new bids right before the auction closes, making it difficult to predict the closing price of the auction ahead of time. This, it is important for bidders to submit their bids with the highest price they are comfortable to pay for the asset_
+It's the auctioneers duty to finish the auction by sending the transaction that calculates the price of the auction on-chain. Though it's also a permissionless process and everyone can do it.
+
+Check out the [guide](https://github.com/gnosis/ido-contracts#settle-auctionss) for doing the price calculation.
+
+#### 4. Bidders: Claiming
+
+After the price has settled, the auctioneer will receive the proceeds of the auction. In contracts the bidder will need to submit an additional transaction to claim their auction proceeds.
```

Comment:
itt should be
``` 
In contrast, ..

```

Reply:
Is there a way to make the images clickable so people can enlarge them? For the images are unusable, they are way too small to read

Reply:
Aside from that everything looks good. 
@josojo I liked the whitelist section

Reply:
Yes, I agree that htis is a necessary feature. I will create an issue